src/web/mjs/engine/Settings.mjs

```diff
@@ -125,6 +125,13 @@ export default class Settings extends EventTarget {
             value: extensions.img
         };
 
+        this.includesComicFile = {
```

Comment:
I suggest to not add a setting for this to prevent setting pollution.
Just always add this sidecar file.